ca-functional/src/test/java/uk/sky/ovp/ca/functional/models/HttpRequest.java

```diff
@@ -0,0 +1,25 @@
+package uk.sky.ovp.ca.functional.models;
+
+import io.restassured.response.Response;
+import org.springframework.stereotype.Component;
+
+@Component
+public class HttpRequest {
+    private Response response;
+
+    public Response getResponse() {
```

Comment:
Looks good and this definitely works! In the future,  you could consider using Lombok and its setter and getter notations that would help reduce the code you write especially if you have multiple class variables.

Reply:
Thanks @zyrondias - we learnt about lombok yesterday, though we wanted to keep the old style for visualisation purposes. :)